monitoring/grafana/NODE.md

```diff
@@ -14,79 +14,55 @@ The dashboard consists of the following sections:
 
 ### Node Health
 
-Row 1:
-* Health Status (gauge bar, ssv+eth1+beacon) \
-  `ssv_node_status{}`, `ssv_eth1_status{}` and `ssv_beacon_status{}`
-* Execution client health (time-series, ok|syncing|disconnected) \
-  `ssv_eth1_status{}`
-* Consensus client health (time-series, ok|syncing|disconnected) \
-  `ssv_beacon_status{}`
-* SSV node health (time-series, up|error|down) \
-  `ssv_node_status{}`
-
-Row 2:
-* ETH1 last synced block (gauge) \
-  `ssv_eth1_last_synced_block{} = <block_number>`
-* ETH1 registry events (gauge bar with counters) \
-  `ssv_eth1_registry_event{name=<event_name>} = <counter>`
+**Row 1:**
+* Health status for the ssv node, eth1 and the beacon node: `ssv_node_status{}` (gauge)
+* Health status for eth1: `ssv_eth1_status{}` (gauge)
```

Comment:
Could you please keep the set of possible states (`ok|syncing|disconnected`, same for other nodes)?